pyxel/core/src/pyxelcore/input.cc

```diff
@@ -7,7 +7,8 @@
 
 namespace pyxelcore {
 
-Input::Input() {
+Input::Input() 
+{
```

Comment:
It is not reasonable to change the indentation style to start `{` on a newline by itself.

It adds extra lines without an increase in readability (I think), it is inconsistent with the rest of the project and is an uncommon style (I _very_ rarely see it across any language PHP, C, JS, Go, Rust).

See also Google's C++ style guide https://google.github.io/styleguide/cppguide.html#Namespaces

```cc
// In the .cc file
namespace mynamespace {

void MyClass::Foo() {
}
}  
```